Workflow/Sources/SubtreeManager.swift

```diff
@@ -404,23 +404,16 @@ extension WorkflowNode.SubtreeManager {
 
 extension WorkflowNode.SubtreeManager {
     struct ChildKey: Hashable {
-        var childType: Any.Type
+        var childTypeID: ObjectIdentifier
         var key: String
 
-        init(childType: (some Any).Type, key: String) {
-            self.childType = childType
+        init(
+            childType: (some Workflow).Type,
+            key: String
+        ) {
+            self.childTypeID = ObjectIdentifier(childType)
             self.key = key
         }
-
-        func hash(into hasher: inout Hasher) {
-            hasher.combine(ObjectIdentifier(childType))
-            hasher.combine(key)
-        }
-
-        static func == (lhs: ChildKey, rhs: ChildKey) -> Bool {
-            lhs.childType == rhs.childType
-                && lhs.key == rhs.key
-        }
```

Comment:
we get these for free if all our stored properties are Hashable

Reply:
you may find [this wrapper](https://github.com/square/Blueprint/blob/main/BlueprintUI/Sources/Internal/Metatype.swift) useful to do the same thing with better debugging/logging